src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -102,15 +102,15 @@ public function getValue($objectOrArray, $propertyPath)
     /**
      * {@inheritdoc}
      */
-    public function setValue(&$objectOrArray, $propertyPath, $value)
+    public function setValue(&$objectOrArray, $propertyPath, ...$value)
```

Comment:
That's a BC break, not allowed per our policy. That also allows calling the method with only two arguments, which wouldn't make sense.
Both issues can be fixed by leaving the signature as is and using `func_get_args()` in the body.

Reply:
Yep and even doing a `public function setValue(&$objectOrArray, $propertyPath, $value, ...$values)` wouldn't make sense in 5.0, as we would need to shift `$value` to `$values` anyway. Staying with a proper `func_get_args()` call is the way to go IMO.

Reply:
I understood your concerns guys. Thanks!

Reply:
Changed to implementation with `func_get_args`. No BC breaks now.